src/restconfig/src/main/java/org/geoserver/rest/catalog/DataStoreFileController.java

```diff
@@ -216,10 +216,12 @@ public ResponseEntity dataStoresGet(
             throw new RestException("No files for datastore " + storeName, HttpStatus.NOT_FOUND);
         }
 
-        try (ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
-                BufferedOutputStream bufferedOutputStream =
-                        new BufferedOutputStream(byteArrayOutputStream);
-                ZipOutputStream zipOutputStream = new ZipOutputStream(bufferedOutputStream)) {
+        try {
```

Comment:
This looks like an unrelated format change.

Reply:
basically this was producing an **REC_CATCH_EXCEPTION)** on QA check.

The definition is available at http://findbugs.sourceforge.net/bugDescriptions.html